migrations/1687269060_addFooterCertificateLinks.ts

```diff
@@ -0,0 +1,105 @@
+import { Client, SimpleSchemaTypes } from "@datocms/cli/lib/cma-client-node";
+
+export default async function (client: Client) {
+  const newFields: Record<string, SimpleSchemaTypes.Field> = {};
+  const newFieldsets: Record<string, SimpleSchemaTypes.Fieldset> = {};
+
+  console.log("Creating new fields/fieldsets");
+
+  console.log('Create fieldset "Certificates" in model "Footer" (`footer`)');
+  newFieldsets["712332"] = await client.fieldsets.create("47595", {
+    title: "Certificates",
+    collapsible: true,
+  });
+
+  console.log(
+    'Create Single-line string field "B corp link" (`b_corp_link`) in model "Footer" (`footer`)'
+  );
+  newFields["9263009"] = await client.fields.create("47595", {
+    label: "B corp link",
+    field_type: "string",
+    api_key: "b_corp_link",
+    validators: { format: { predefined_pattern: "url" } },
+    appearance: {
+      addons: [],
+      editor: "single_line",
+      parameters: { heading: false },
+    },
+    default_value: "",
+    fieldset: newFieldsets["712332"],
+  });
+
+  console.log(
+    'Create Single asset field "Bcorp logo" (`bcorp_logo`) in model "Footer" (`footer`)'
+  );
+  newFields["9338640"] = await client.fields.create("47595", {
+    label: "Bcorp logo",
+    field_type: "file",
+    api_key: "bcorp_logo",
+    validators: {
```

Comment:
I think the logos should be required (DDA as well)

Reply:
✅